programs/wen_new_standard/src/instructions/mint/create.rs

```diff
@@ -49,13 +43,12 @@ pub struct CreateMintAccount<'info> {
         mint::token_program = token_program,
         mint::decimals = 0,
         mint::authority = authority,
-        mint::freeze_authority = manager,
```

Comment:
need this to stay manager so we can add freeze instruction to WNS

Reply:
resolved above